openlcs/libs/corgi.py

```diff
@@ -612,7 +612,7 @@ def get_component_instance_from_nvr(self, nvr):
         raise ValueError(err_msg)
 
     def collect_components_from_subscriptions(self,
-                                              subscriptions,
+                                              subscription,
```

Comment:
Similar here - original params is plural. if it's updated to collect one at a time, do you think func name should be changed as well?

Reply:
Updated.